File: BeachVolleyball.Server/ClientApp/src/components/Tournaments/Match.js
import React from "react";
import styled from "styled-components";
import Team from "./Team";

const Match = ({ team1, team2 }) => (
  <MatchContainer>
    <Team {...team1} />
    <VsText>נגד</VsText>
    <Team {...team2} />
  </MatchContainer>
);

const MatchContainer = styled.div`
  display: flex;
  flex-direction: row;
  align-items: center;

  & > * {
    margin-left: 1rem;
  }
`;

const VsText = styled.p`
  font-size: 1.2rem;
`;

export default Match;

File: BeachVolleyball/Match.cs
//---------------------------------------------------------------------------------------------------------------------
// <copyright file="Match.cs" company="Microsoft">
//     Copyright (c) Microsoft. All rights reserved.
// </copyright>
//---------------------------------------------------------------------------------------------------------------------

namespace BeachVolleyball
{
    public class Match
    {
        public Match(Team team1, Team team2)
        {
            this.Team1 = team1;
            this.Team2 = team2;
        }

        public Team Team1 { get; private set; }

        public Team Team2 { get; private set; }
    }
}
